src/patterns-hmrc-govuk-team/forms-detailed-guide/index.njk

```diff
@@ -19,17 +19,22 @@ Use this pattern if users need to read detailed guidance before using the form,
 </ul>
 
 <p class="govuk-body">
-Before you create the new detailed guide, check if any of the information is already somewhere else, for example in mainstream content or a specialist guide. If it is, do not repeat it, insert a link instead.
+Before you create the new detailed guide or add the form pattern to an existing guide, check if any of the information is already somewhere else, for example in mainstream content or a specialist guide. If it is, do not repeat it, insert a link instead.
 </p>
 
 <p class="govuk-body">
-Types of form that can be included in this format are: 
+If it is, do not repeat it. Add a link and journey to that content instead.
+</p>
+
+<p class="govuk-body">
+Types of form that you can use in a detailed guide include the following: 
 </p>
 
 <ul class="govuk-list govuk-list--bullet">
-<li>Interactive guidance forms — these are added as external links. The URL will begin with https://www.tax.service.gov.uk/guidance</li>
-<li>PDFs that can be completed on-screen (interactive) — these are added as file attachments and require Adobe to open and complete</li>
-<li>PDFs that cannot be completed on screen (not interactive) — these are added as file attachments and do not require Adobe to open</li>
+<li>Online services — These are built by service teams on the tax platform behind the Government Gateway, accessed with a green button with a URL, starting with https://www.taxservices.gov.uk/</li>
```

Comment:
I've never seen `https://www.taxservices.gov.uk/` so I'm guessing it should actually be:
```suggestion
<li>Online services — These are built by service teams on the tax platform behind the Government Gateway, accessed with a green button with a URL, starting with https://www.tax.service.gov.uk/</li>
```